e2e/cypress/support/commands.js

```diff
@@ -71,7 +71,7 @@ Cypress.Commands.add('createProject', (projectName, projectDescription) => {
   });
 
   log.snapshot('open-modal');
-  cy.get('[data-test-id="create-new-project-button"]').click({ force: true });
+  cy.get('[data-test-id="create-new-project-button"]').scrollIntoView().click();
```

Comment:
why this change? I don't remember why I had to use the `force: true` but the modal shouldn't need any scrolling right?